content/en/docs/Core concepts/sharding.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
I put here the sentences I extracted from the other file after comments, just to avoid losing them